api/test/factories/submissions.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :submission do
+    response { Faker::Creature::Animal.name }
+    status { 0 }
```

Comment:
You can (and should) use the enum value here:

 ```suggestion
    status { :working }
```